Example/Tests/Tests.m

```diff
@@ -168,6 +168,21 @@
         label.text = @"Hello there, this is some pretty long text ey? Maybe it'll wrap.";
         expect(label).to.haveValidSnapshot();
     });
+
+    describe(@"preffered max layout width", ^{
+        it(@"sets it to device width - margin on init", ^{
+            label = [[ARLabel alloc] init];
+            expect(label.preferredMaxLayoutWidth).to.equal(280);
+        });
```

Comment:
~~For the feature? Yes, AL cannot figure out how many lines it should show without a hard width constraint~~

Maybe not, there's hints that this has changed since iOS8 ( [SO](http://stackoverflow.com/questions/25398312/automatic-preferred-max-layout-width-is-not-available-on-ios-versions-prior-to-8) )

This'd require _careful_ testing